shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -259,6 +264,13 @@ void AccessibilityBridge::AddSemanticsNodeUpdate(
   for (const auto& value : update) {
     size_t this_node_size = sizeof(fuchsia::accessibility::semantics::Node);
     const auto& flutter_node = value.second;
+    // We handle root update separately in GetRootNodeUpdate.
+    // TODO(chunhtai): remove this special case after we remove the inverse
+    // view pixel ratio transformation in scenic view.
+    if (flutter_node.id == kRootNodeId) {
+      root_flutter_semantics_node_ = flutter_node;
+      continue;
```

Comment:
Is this right? We don't want to set the ID or flags or whatever else might be present on the root node?

Reply:
They will be set in GetRootNodeUpdate